pkg/probe/system_sdn_connector.go

```diff
@@ -0,0 +1,62 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type SystemSdnConnectorResults struct {
+	Name       string `json:"name"`
+	Type       string `json:"type"`
+	Status     string `json:"status"`
+	Updating   bool   `json:"updating"`
+	LastUpdate int    `json:"last_update"`
+}
+
+type SystemSdnConnector struct {
+	Results []SystemSdnConnectorResults `json:"results"`
+	VDOM    string                      `json:"vdom"`
+}
+
+func probeSystemSdnConnector(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		SdnConnectorsStatus = prometheus.NewDesc(
+			"fortigate_system_sdn_connector_status",
+			"Status of SDN connectors (0=Disabled, 1=Down, 2=Unknown, 3=Up, 4=Updating)",
+			[]string{"vdom", "name", "type"}, nil,
+		)
+		SdnConnectorsLastUpdate = prometheus.NewDesc(
+			"fortigate_system_sdn_connector_last_update",
```

Comment:
Rename to `fortigate_system_sdn_connector_last_update_seconds` as per https://github.com/bluecmd/fortigate_exporter/blob/master/CONTRIBUTING.md#naming-convention.

Description should also reference that this is epoch time.